src/System Application/Test Library/AI/src/CopilotTestLibrary.Codeunit.al

```diff
@@ -29,6 +29,18 @@ codeunit 132932 "Copilot Test Library"
         CopilotCapability.RegisterCapability(Capability, '');
     end;
 
+    procedure RegisterCopilotCapabilityForSpecificAppId(Capability: Enum "Copilot Capability"; AppId: Guid)
+    var
+        CopilotSettings: Record "Copilot Settings";
+    begin
+        CopilotCapability.RegisterCapability(Capability, '');
```

Comment:
```suggestion
        RegisterCopilotCapability(Capability);
```

Use as overload. First overload should be changed but that's fine for now.

Reply:
Fixed